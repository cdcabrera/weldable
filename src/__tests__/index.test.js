const weldable = require('../index');

describe('weldable', () => {
  it('should return specific properties', () => {
    expect(weldable).toMatchSnapshot('specific properties');
  });
});
