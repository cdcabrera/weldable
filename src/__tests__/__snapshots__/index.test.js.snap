// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`weldable should return specific properties: specific properties 1`] = `
{
  "OPTIONS": {
    "_set": undefined,
    "contextPath": "../__fixtures__",
  },
  "weldable": [Function],
}
`;
